src/EventLoop.hs

```diff
@@ -50,13 +50,14 @@ eventFromPullRequestPayload payload =
     number = PullRequestId payload.number
     title  = payload.title
     author = payload.author
+    body   = payload.body
     branch = payload.branch
     sha    = payload.sha
     baseBranch = Github.baseBranch (payload :: PullRequestPayload)
   in
     case payload.action of
-      Github.Opened      -> Logic.PullRequestOpened number branch baseBranch sha title author
-      Github.Reopened    -> Logic.PullRequestOpened number branch baseBranch sha title author
+      Github.Opened      -> Logic.PullRequestOpened number branch baseBranch sha title author body
+      Github.Reopened    -> Logic.PullRequestOpened number branch baseBranch sha title author Nothing
```

Comment:
While this does solve hoff from parsing merge commands from PR descriptions when it is closed and reopened. This way of doing it is a bit hidden and implicit. 

Can you add a descriptive comment on why we remove the PR body from the event, how this affects the parsing of merge commands in descriptions, and why we want to avoid that behavior on PR reopens.